CHANGES.rst

```diff
@@ -210,6 +210,8 @@ astropy.table
   found in the input data, and any missing key/value pairs are turned into
   missing data in the table. [#9425]
 
+- Prevent unecessary ERFA warnings when indexing by ``Time`` columns. [#9545]
```

Comment:
does this really need a changelog entry?

Reply:
Well it's an improvement in the user experience :) But I don't mind either way.

Reply:
🤷‍♀ - we've removed tons of warnings from all over, but I don't mind leaving it in either.

Reply:
Ok just to be clear this is a user-facing warning (not just tests) and the ERFA warnings are particularly annoying and cryptic. I agree it's probably overkill to mention here though.